plugins/experimental/ssl_session_reuse/src/config.cc

```diff
@@ -62,7 +62,7 @@ Config::loadConfig(const std::string &filename)
     size_t n = info.st_size;
```

Comment:
We should be assigning ts::NO_FD (-1) here instead of 0.  Also, we should check to see if the open failed and return failure right away instead of calling fstat with an invalid fd.

Reply:
Yeah, I like this way better. Done.